.circleci/config.yml

```diff
@@ -77,10 +35,84 @@ jobs:
           command: |
             cd drupal/web && php core/scripts/run-tests.sh --verbose --sqlite --module elasticsearch_helper
 
+jobs:
+  build:
+    docker:
+      - image: wunderio/circleci-codeception-base
+
+    steps:
+      - run:
+          name: Create Drupal
+          command: |
+            composer create-project drupal/recommended-project:8.8.6 drupal
+            cd drupal
+            COMPOSER_MEMORY_LIMIT=-1 composer require drupal/core-dev:^8
+            COMPOSER_MEMORY_LIMIT=-1 composer require drush/drush:9
+            COMPOSER_MEMORY_LIMIT=-1 composer require elasticsearch/elasticsearch:~6.0
```

Comment:
I guess we should use elasticsearch/elasticsearch:~6.0 version for es-6 testing and elasticsearch/elasticsearch:~7.0 version for es-7 testing.